test/test_opaque_obj.py

```diff
@@ -90,7 +90,7 @@ def pop_impl_fake(q: torch._C.ScriptObject) -> torch.Tensor:
             # This is not accurate since the queue could have tensors that are
             # not rank 1
             ctx = torch._custom_op.impl.get_ctx()
-            u0 = ctx.create_unbacked_symint()
+            u0 = ctx.new_dynamic_size()
```

Comment:
Add a description to the PR? I'm not sure what a "non-effectful scriptobjs" is right now